db/migrate/20170326022716_add_article_image_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddArticleImageToArticles < ActiveRecord::Migration[5.0]
+  def change
+    add_column :articles, :article_image, :string
```

Comment:
Also, I like to treat URLs as `:text` instead of `:string` in case they grow really long.